core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
The difference (I want to set after this PR) is, `Constants.java` serving constants determined in compile time, and `RuntimeConstants` serving constants determined once the process is launched.